pkg/operations/operations_aws.go

```diff
@@ -130,33 +152,48 @@ type awsConnection struct {
 }
 
 var (
-	awsDefaultSession      *session.Session
+	awsDefaultSessions     map[string]*session.Session = map[string]*session.Session{}
 	awsDefaultSessionMutex sync.Mutex
 )
 
-func getAWSSession(awsRegion, awsAccessKey, awsSecretKey, token string) (*session.Session, error) {
+func getAWSSession(awsRegion, awsAccessKey, awsSecretKey, awsToken, awsProfile string, testSession bool) (*session.Session, error) {
```

Comment:
Most of the args are obvious by name, but a comment for what testSession is for would make this clearer. And maybe validate would be a better name for it?